frontend/public/components/notification-drawer.tsx

```diff
@@ -105,25 +108,47 @@ const getAlertNotificationEntries = (
     : [];
 
 const getUpdateNotificationEntries = (
+  cv: ClusterVersionKind,
   isLoaded: boolean,
-  updateData: ClusterUpdate[],
   isEditable: boolean,
   toggleNotificationDrawer: () => void,
-): React.ReactNode[] =>
-  isLoaded && !_.isEmpty(updateData) && isEditable
+): React.ReactNode[] => {
+  const updateData: ClusterUpdate[] = getSortedUpdates(cv);
+  const currentChannel = cv?.spec?.channel;
+  const currentPrefix = splitClusterVersionChannel(currentChannel)?.prefix;
+  const similarChannels = getSimilarClusterVersionChannels(currentPrefix);
+  const newerChannel = getNewerClusterVersionChannel(similarChannels, currentChannel);
+  const newerChannelVersion = splitClusterVersionChannel(newerChannel)?.version;
+  return isLoaded && !_.isEmpty(updateData) && isEditable
```

Comment:
I think `isLoaded` can be removed. The behavior should be the same with no other changes.